src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -96,96 +90,96 @@ import './ContentExplorer.scss';
 const GRID_VIEW_MAX_COLUMNS = 7;
 const GRID_VIEW_MIN_COLUMNS = 1;
 
-type Props = {
-    apiHost: string,
-    appHost: string,
-    autoFocus: boolean,
-    canCreateNewFolder: boolean,
-    canDelete: boolean,
-    canDownload: boolean,
-    canPreview: boolean,
-    canRename: boolean,
-    canSetShareAccess: boolean,
-    canShare: boolean,
-    canUpload: boolean,
-    className: string,
-    contentPreviewProps: ContentPreviewProps,
-    contentUploaderProps: ContentUploaderProps,
-    currentFolderId?: string,
-    defaultView: DefaultView,
-    features: FeatureConfig,
-    fieldsToShow?: FieldsToShow,
-    initialPage: number,
-    initialPageSize: number,
-    isLarge: boolean,
-    isMedium: boolean,
-    isSmall: boolean,
-    isTouch: boolean,
-    isVeryLarge: boolean,
-    language?: string,
-    logoUrl?: string,
-    measureRef?: Function,
-    messages?: StringMap,
-    metadataQuery?: MetadataQuery,
-    onCreate: Function,
-    onDelete: Function,
-    onDownload: Function,
-    onNavigate: Function,
-    onPreview: Function,
-    onRename: Function,
-    onSelect: Function,
-    onUpload: Function,
-    previewLibraryVersion: string,
-    requestInterceptor?: Function,
-    responseInterceptor?: Function,
-    rootFolderId: string,
-    sharedLink?: string,
-    sharedLinkPassword?: string,
-    sortBy: SortBy,
-    sortDirection: SortDirection,
-    staticHost: string,
-    staticPath: string,
-    theme?: Theme,
-    token: Token,
-    uploadHost: string,
-};
+export interface ContentExplorerProps {
```

Comment:
I see that many props are changed to optional. Is the intention that only the `token` prop is needed? Can we verify that this is the only prop required to get Explorer to run?

Reply:
thats what it seems like, only the `rootFolderId` and `token` is needed, thats pretty much whats in the basic storybook for contentexplorer,